test/stress_tests.py

```diff
@@ -102,6 +102,30 @@ def g(n):
     assert ray.services.all_processes_alive()
 
 
+def test_submitting_many_actors_to_one(ray_start_sharded):
```

Comment:
Does this test fail before this PR?

Reply:
It didn't on my laptop, but it was on EC2, which was how we initially found this issue. Travis is failing on this test right now, which unfortunately might be because we're trying to start too many processes.

Reply:
I'll try lowering the number of processes, and if Travis is still failing, I'll remove the test.